src/coprocessor/mod.rs

```diff
@@ -127,6 +137,17 @@ impl ReqContext {
     ) -> Self {
         let deadline = Deadline::from_now(max_handle_duration);
         let bypass_locks = TsSet::from_u64s(context.take_resolved_locks());
+        let mut min = match ranges.first().as_ref() {
+            Some(range) => range.start.clone(),
+            None => vec![],
+        };
+        let mut max = match ranges.last().as_ref() {
+            Some(range) => range.end.clone(),
+            None => vec![],
+        };
+        if min > max {
```

Comment:
Is it possible that `min` is not empty but `max` is empty?
Is it possible that `min` and `max` are both empty?